whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/DummifierRC.java

```diff
@@ -1,17 +1,56 @@
 package net.ripe.db.whois.common.rpsl;
 
+import com.google.common.base.Splitter;
 import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import net.ripe.db.whois.common.rpsl.transform.FilterChangedFunction;
+import org.apache.commons.lang.Validate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
 
+import java.util.Iterator;
 import java.util.List;
+import java.util.Set;
+
+import static net.ripe.db.whois.common.rpsl.AttributeType.ABUSE_MAILBOX;
+import static net.ripe.db.whois.common.rpsl.AttributeType.ADDRESS;
+import static net.ripe.db.whois.common.rpsl.AttributeType.AUTH;
+import static net.ripe.db.whois.common.rpsl.AttributeType.E_MAIL;
+import static net.ripe.db.whois.common.rpsl.AttributeType.FAX_NO;
+import static net.ripe.db.whois.common.rpsl.AttributeType.IRT_NFY;
+import static net.ripe.db.whois.common.rpsl.AttributeType.MNT_NFY;
+import static net.ripe.db.whois.common.rpsl.AttributeType.NOTIFY;
+import static net.ripe.db.whois.common.rpsl.AttributeType.PERSON;
+import static net.ripe.db.whois.common.rpsl.AttributeType.PHONE;
+import static net.ripe.db.whois.common.rpsl.AttributeType.REF_NFY;
+import static net.ripe.db.whois.common.rpsl.AttributeType.UPD_TO;
 
 @Component
-public class DummifierRC extends DummifierCurrent {
+public class DummifierRC implements Dummifier {
```

Comment:
At the risk of *more* confusion, could you rename to `DummifierCurrent` ? *This* is now the current (default) dummifier.

Reply:
To answer my own question: in fact `DummifierNrtm` is used to create the split files. Leave this class name as-is.